WalletWasabi.Fluent/Views/Wallets/Settings/WalletCoinJoinSettingsView.axaml

```diff
@@ -28,7 +29,27 @@
         <Button Theme="{StaticResource AccentButton}" Content="Change" Command="{Binding SelectCoinjoinProfileCommand}" DockPanel.Dock="Right" />
         <TextBlock Text="{Binding SelectedCoinjoinProfileName}" VerticalAlignment="Center" MinWidth="120" Margin="10 0 10 0" />
       </DockPanel>
+
+      <DockPanel>
+        <TextBlock Text="Output Wallet" />
+        <PathIcon Data="{StaticResource warning_regular}" Foreground="{DynamicResource WarningMessageForeground}" DockPanel.Dock="Right" Margin="10,0,0,0" Height="20">
+          <ToolTip.Tip>
+            <TextBlock>
+              - Only loaded wallets can be selected.<LineBreak /><LineBreak />
+              - Coins from the CoinJoin transaction will be received by the output wallet. However, this setting resets after Wallet Wasabi restart.<LineBreak /><LineBreak />
+              - Until coinjoin is running you can't set the output wallet.
```

Comment:
```suggestion
              - Output wallet cannot be changed when a coinjoin is in progress
```

Or `Output wallet can only be changed if coinjoin is not in progress", but the current sentence is incorrect

Reply:
done